MAINTAINING.md

```diff
@@ -25,6 +25,23 @@ Once you have your build environment set up and the WebRTC source downloaded:
     # Put it in our frameworks search path
     mv src/webrtc/ios_libs_out/WebRTC.framework ../Signal-iOS/Carthage/Builds
 
+    # The specific set of commands that worked for me were somewhat different.
```

Comment:
No point in keeping the previous incorrect instructions. Could you delete them? Otherwise LGTM.